tests/app/views/test_suppliers.py

```diff
@@ -1480,69 +1463,53 @@ def supplier_framework_interest(self, framework_slug, update):
                 }),
             content_type='application/json')
 
-    def test_get_supplier_framework_info_non_existent_by_framework(self):
-        response = self.client.get(
-            '/suppliers/0/frameworks/g-cloud-5')
-
-        assert response.status_code == 404
-
-    def test_get_supplier_framework_info_non_existent_by_supplier(self):
-        response = self.client.get(
-            '/suppliers/123/frameworks/g-cloud-4')
-
-        assert response.status_code == 404
-
-    def test_adding_supplier_has_passed(self, open_dos_framework):
+    def test_adding_supplier_has_passed(self, supplier_framework):
         response = self.supplier_framework_interest(
-            'digital-outcomes-and-specialists',
+            supplier_framework,
             update={'onFramework': True}
         )
         assert response.status_code == 200
         data = json.loads(response.get_data())
-        assert data['frameworkInterest']['supplierId'] == 0
-        assert data['frameworkInterest']['frameworkSlug'], 'digital-outcomes-and-specialists'
+        assert data['frameworkInterest']['supplierId'] == supplier_framework['supplierId']
+        assert data['frameworkInterest']['frameworkSlug'] == supplier_framework['frameworkSlug']
         assert data['frameworkInterest']['onFramework'] is True
-        assert data['frameworkInterest']['agreementReturned'] is False
```

Comment:
Wee losing a lot of asserts here... Is it worth adding these things to the fixture and keeping the asserts in? Or is that a bit pointless?